### DISCUSSION TITLE
Hoe wordt berging op maaiveld meegenomen binnen Ribasim?

### DISCUSSION THREAD

**Discussion by SnippenE** [Deltares/Ribasim#416] [Q&A] [answered]
Op basis van stationaire sommen om Qh-relatie op te bouwen verdwijnt de invloed van berging die in hydraulisch model mogelijk wel was ingebouwd (Connection Nodes with Storage). De afvlakkende werking door berging op maaiveld komt daarmee niet terug in Qh-relatie.

**Comment by visr** [comment, marked as answer]
Het klopt dat op deze manier de berging niet in de Qh relaties meekomt. Tot nu toe is de focus meer op waterverdeling dan hoogwater scenarios. Door de profielen van de Basins boven de sloten door te trekken kan je meer water bergen, maar andere aannames zijn dan wellicht niet meer geldig, omdat er bijvoorbeeld nieuwe doorstroom paden in het echt ontstaan die niet in het model zitten.
